src/applications/vaos/components/AfterVisitSummary.jsx

```diff
@@ -13,12 +14,22 @@ import Section from './Section';
 import { revokeObjectUrls } from '../utils/avs';
 import useAmbAvs from './hooks/useAmbAvs';
 
-function handleClick() {
+function handleLegacyAvsClick() {
   recordEvent({
     event: `${GA_PREFIX}-after-visit-summary-link-clicked`,
   });
 }
 
+function handleOhAvsPdfClick(pdfCount) {
+  recordEvent({
+    event: `${GA_PREFIX}-after-visit-summary-link-clicked`,
+    'avs-type': 'oh-pdf',
```

Comment:
Does this field need additional setup from the Analytics & Insights team? I think custom fields need some mapping in GA4 to be useful. If you want to avoid that process, we should just use a different event name.